src/popup/router/components/Platforms.vue

```diff
@@ -113,32 +131,34 @@ export default {
 
     .mobile-app,
     .extension {
-      padding: 15px 0;
+      padding-top: 15px;
 
       div {
         display: flex;
-        justify-content: space-around;
+        justify-content: center;
 
         + div {
           padding-top: 15px;
+          padding-bottom: 12px;
         }
 
         a {
-          padding: 5px;
-          border-radius: 10px;
-          height: 52px;
-          width: 52px;
+          &:first-of-type {
+            margin-right: 32px;
```

Comment:
I think we can avoid this stuff by using 
```
display: flex;
justify-content: space-evenly;
```
in parent

Reply:
Yes, we can, but it wouldn't be pixel perfect.

Reply:
I thought we already discussed it during the team meeting. 
Current designs have been done using standart screen size and ratio of an Android device (360x640px).
Furthermore:
_"High-density screens can be a bit confusing, a pixel isn't just a pixel anymore: the actual pixel count is a multiple of the declared pixel count.
That particular resolution is common because it's the simplest math for HD formats:
360x640 pixels, at 200% DPI, is 720x1280 -- which is the viewport size of the HDTV standard (aka 720p).
At 300% DPI it is 1080x1920 -- which is the 1080p standard."_

The need to decrease the viewport height from 640 to 600 px appears because of the Google Chrome (and chromium based browsers) limitation of viewport max-height to 600px. As it turned out none of us was aware of this limitation (I have asked before starting the designs). Currently it will be time consuming to adjust all the screens and the prototype interactions for just 40px decrease of viewport height. At the same time it won't be so good idea as current size is better for the native apps. 

My suggestion will be to apply "display: flex; justify-content: space-evenly;" suggested above but only for chromium-based browsers. The difference of 40px distributed evenly in this case won't be a problem. For web-based wallet, apps and non chromium browsers we may still have 640px height (as mentioned above it's very standard aspect ratio of the viewport).

On the welcome screens it will be better to diminish the size of the margins between the spinning animation and the rest of the content by 20px on the top and another 20px on the bottom. It also can be done quickly with a browser-specific extra line of CSS. This is the most obvious and quick solution here. 

For the screens where we already have scrollable area this won't be a problem.

Reply:
Fully agree with Tsvetan here!